### ISSUE TITLE
[Feature Request] add buy me a coffee at the bottom of menu

### ISSUE THREAD

**Issue by hcgiub001** [Tikipeter/tikipeter.github.io#118]
As a token of gratitude I'm sure their are plenty of people who want to show their appreciation for your work, me included. Could you add ways to make this possible. As long as I'm using Fenlight, I want to give a donation once a year. I think Developers are taken for granted and this culture in Kodi community that we expect developers to do this and not get compensated is outdated, even if people pay $1 a month , you have probably thousands using Fenlight , so it could help if those that are happy to support your work are able to. 
I've seen some apps already have ways to 

realizer has this at the bottom
![Screenshot (545)](https://github.com/user-attachments/assets/647dba1e-8fe3-41f3-afd9-63dc30b8abe0)

![Screenshot (546)](https://github.com/user-attachments/assets/30ab736a-2c8a-422c-bbea-ea95e3b2cf91)

you could get an affiliate link for real-debrid and people can use the affiliate link and they can support by just using affiliate links, anytime someone installs Fenlight you could show the support & donations page to let them know of it.

I'll also like a buy me a bear account so I can donate directly

Even if you stop developement and keep your addon for maintenance , we still benefit from your code on daily basis.

I think all developers should do this. those that want to donate can and you can get support by people using your affiliate links too. get one with ipvanish like realizer dev, if people want an vpn they have an option their too.
This is more of a demand than a request 

On buy me a coffee you have two payment methods, monthly or yearly. monthly payments of like $3 is something I'd like to donate for a beer. one beer a month (I'm cheap)

**Comment by rizreflects** [comment]
I'm sure Peter will probably say no to this, as I remember someone asking something similar in the past, but just thought I'd chime in this time and say +1 from me. One beer a week? Hell, I'll send him a whiskey a day for all the work he puts in. Along with an Addons4Kodi baseball cap.

**Comment by hcgiub001** [comment]
> I'm sure Peter will probably say no to this, as I remember someone asking something similar in the past, but just thought I'd chime in this time and say +1 from me. One beer a week? Hell, I'll send him a whiskey a day for all the work he puts in. Along with an Addons4Kodi baseball cap.

Even an affiliate link would be great, so people can use the real-debrid affiliate link. But there are legitimate concerns about being in trouble with the law. Even though the addon doesn't provide any pirated content.

I'm sure Peter has heard of Exodus and Lambda
I was out of the scene when it happened but they got a law suit against him and he wasn't even taking donations

"“Lambda,” who was involved with the XBMC/Kodi community. Lambda was known for creating the popular Kodi add-on called “Exodus,” which allowed users to stream movies and TV shows.

[In 2017, Lambda, whose real name is Dimitrios Papadimitriou, faced legal issues due to the nature of the add-on, which facilitated access to copyrighted content without proper authorization](https://forums.moneysavingexpert.com/discussion/6174405/ever-sued-a-new-build-developer-won)[1](https://forums.moneysavingexpert.com/discussion/6174405/ever-sued-a-new-build-developer-won). This led to significant legal pressure from copyright holders and anti-piracy organizations."

I used Microsoft co-pilot to give me this, try searching on google or yandex and I can find zero mention of this, I've never seen this before , it's like it never happened!
Any donations or even affiliate links will be seen by authorities, and Fenlight being as popular as it is, (same with Exodus in it's peak days) then I won't be surprised they clamp down on Tikipeter to make an example of him.

**Comment by rizreflects** [comment]
Exodus used to scrape for free links available online from what I remember of it, alongside debrid which was later added. So very different to Fen, and probably the reason the dev got into trouble. Fens more comparable to stremio, or even kodi itself in that regard.

**Comment by hcgiub001** [comment]
I forgot I didn't use debrid in those days and I agree

**Comment by Tikipeter** [comment]
I'm almost certain that AI generated text is incorrect. The owner of TVAddons and the dev of an addon featuring live streams were sued as a result of all the happenings of that time period. Lamdba was already "gone" for adding some code in Exodus that could be interpreted as malicious.

Also, there were some OG devs back in the day who decided to break free of TVAddons and form their own group where they would charge for Kodi addons. They were very good devs, but their endeavour failed.

In life, money is not really that important to me. I have what I need, and don't really wish for more at this time. Adding some kind of charity link would not be an incentive to continue or leave the scene really. Perhaps some, maybe even most, devs would think differently, but that hasn't been the case yet and it still might not be.

The legal ramifications may be none or may be important. No one knows. The prevailing thought is to not accept money for providing an addon like this, so that is how it has always been. The issue is the addons aren't doing anything wrong either way, but if they want to come after you that fact won't stop them. A recent dev had a letter sent from a production company threatening legal action, and that addon provided no live links to the movie in question.

Long story short, it's not something I would ever do, and if I did do it it wouldn't cause the outcome I think you expect it would i.e. to increase motivation. It's just not the way my brain works with regard to money.

There are, however, plenty of worthwhile places to place some money when it comes to the Kodi community. Definitely the Kodi devs themselves, but also TMDb, Trakt, Fanart.tv etc. These services allow Kodi to function. There would be no Kodi or Kodi addons without these free services. Also look into whether the dev of the skin you are using has support information where some money could be thrown their way as well. They don't have the worry of whether that causes greater exposure to themselves.

**Comment by hcgiub001** [comment]
I was actually around when this happened. My name was MediaPi, I was (and still am) a mischievous person. We had single addons, no real-debrid. At that time I use to communicate with dev's at that time and think of ideas. I've talked to great devs like Spoyser (SuperFavourites) highway (kissanime, kisscartoon, I actually asked him to create kissanime and spent so long asking him to add things from the site, it turned into an awesome addon but the site got harder to scrape)

Lambda came and changed the game when we can search multiple sites, it wasn't done before he came. I even talked to him asking for features, not sure what they were. Then I left the scene and I heard about Lambda, he was getting some threats from a site, I forgot why, so he put in some code that every time someone made a search it ping that site once (or twice).

Imagine how many people were using Exodus back then, it was the most popular addon, nothing came close. It probably recked the site. Someone checked his code and asked why is this here.

The community turned on him so fast. The text I gave about him being sued I learnt myself from some site back then. their was no ai back then. I remember clearly thinking how unfair to harass someone that got nothing out of it, no financial gain. I'm curious to see if I can find any mention of this.

I don't even care if you stop adding features and just maintained it. At least consider adding a referral link. This isn't for me to say stay in kodi. In fact I think you should think of yourself and have an exit point. All your peers have left already. It's working great as it is and doesn't really need anything. 

I just find it unfair and referral link will not cost anyone anything and after you can use the money to do what you want with it. Even give it to the kodi foundation or treat your loved one with a holiday.

If you decide to keep status quo, close the issue, but I would love it if you reconsidered.
just because no developers was taking donations doesn't mean no dev should, and some are already like I've shown.

Anyway you're a great man, the community doesn't deserve you.
